Resolutions/ChildOfLight_Resolution/rules.txt

```diff
@@ -89,7 +89,6 @@ $gameHeight = 720
 [TextureRedefine] # TV Resolution
 width = 1280
 height = 720
-formats = 0x011,0x01a,0x001
```

Comment:
Oh, I forgot to add this one as a comment. Same thing as the other formats. Sorry!

Reply:
Okay I've added back that one as well, that should be it, right?

Reply:
I still don't agree with adding the padded half-res TextureRedefine and you didn't re-add all the format filters 😅

Reply:
Okay, that's everything. Can this be merged now?